### ISSUE TITLE
Latest commit breaks the code

### ISSUE THREAD

**Issue by wlbr** [jamesmoriarty/gohack#30]
Hi, your latest commit/the cleanup/refactoring has broken the code. Neither the test against the test executable works, nor a connect to the "real" csgo.
Anyway, checking out the last version helps, but ...

Regards,
Michael

**Comment by jamesmoriarty** [comment]
Thanks for reporting this. Will revert once I get back from holidays within the week.

**Comment by jamesmoriarty** [comment]
Thanks for reporting this. I've confirmed tonight (be it a bit late)

`head`

```
INFO[0000] - Address                                     value=0x24b60000
INFO[0000] - OffsetForceJump                             value=0x24b60000
INFO[0000] - OffsetPlayer                                value=0x905a4d
INFO[0000] - OffsetPlayerFlags                           value=0x905a4d
```

`git checkout head~1`

```
INFO[0001] GetFromProcessName csgo.exe                   pid=11180
INFO[0001] OpenProcess 11180                             handle=980
INFO[0001] - Address                                     value=0x24b60000
INFO[0001] - OffsetForceJump                             value=0x29dabe84
INFO[0001] - OffsetPlayer                                value=0x58184910
INFO[0001] - OffsetPlayerFlags                           value=0x58184a14
```

You can see its wrong just by the look of the addresses.

**Comment by wlbr** [comment]
Yes, that's exactly how I found out that there is the old, working version. Luckily there was still an old binary available, otherwise, I would not have tested the old code. Unfortunately, though the refactored code is much nicer, I a using the old revision for my playground.

If you are interested: I created a file with all the key encodings as constants. Nothing special, just manual work, not worth a pullrequest. [https://wlbr.de/assets/keys.go](https://wlbr.de/assets/keys.go)
